### ISSUE TITLE
Include all previous git history

### ISSUE THREAD

**Issue by idbrii** [john-orr/renamer.vim#1]
Hi!

It's me again. I love your plugin (and I've emailed you patches in the past). Welcome to github!

I've been tracking your plugin on vim.org in github and I have git history for all previous versions.

Would you consider pulling in my history?
https://github.com/idbrii/renamer.vim/commits/qpkorr-master

If I `git diff f05f240 31a1e39` (your repo at master and mine at qpkorr-master) then there are no differences, but it has more history (to see how bugs were introduced etc).

You can pull in my history like this:

```
# checkout another branch to preserve what you haven't pushed. That commit
# is the master in your branch at the time I pulled in your changes.
git checkout -b history f05f240
git remote add idbrii git@github.com:idbrii/renamer.vim.git
git fetch idbrii
# see that there are no significant differences
git diff history idbrii/qpkorr-master
# This is the only "dangerous" part, but we're in a new branch, so it's safe.
git reset --hard idbrii/qpkorr-master
# Now if you have any new changes since I pushed, you can reapply them on top.
git checkout master
# Check if you've made changes
git log f05f240..master
# If you have changes, rebase on top. If you get any merge conflicts,
# you can skip them since you're rebasing on top of identical code.
git rebase history
# If you don't have any changes, update your master and cleanup
git reset --hard history
git branch -d history
```

Then you'll have history going back to v1.0

And then I can send you some pull requests for bug fixes.

(_Edit_: I inserted some of my patch history that you've accepted before the 2.1 release in my branch and renamed it to qpkorr-master. I've added more information for rebasing your history.)

**Comment by idbrii** [comment]
I edited my original instructions a bit. For simplicity, if you just want to grab my history and haven't made any recent changes, you can just do this:

```
git remote add idbrii git@github.com:idbrii/renamer.vim.git
git fetch idbrii
git reset --hard idbrii/qpkorr-master
```

Also, I think once you do that, github will realize that our two repos (and everyone else who cloned vim-scripts) are the same project and we can automatically do pull requests.

**Comment by idbrii** [comment]
Resolved with 31a1e39d40f16778da49e1a4a97afec98beff8b6
